cmdutil/closureLogic.go

```diff
@@ -5,7 +5,7 @@ import (
 	"io"
 	"os"
 
-	"github.com/dachad/tcpgoon/tcpclient"
+	"github.com/dachad/tcpgoon/mtcpclient"
```

Comment:
:clap: I was worried about consuming an "inner package" from the most external one